### ISSUE TITLE
how CTFs are reported

### ISSUE THREAD

**Issue by rmarabini** [I2PC/scipion#622]
CTF metadata contains the columndefocusRation (that is, defocusU/defocusV). In my opinion this magnitude should be replaced by defocusU - defocusV since the limitation on resolution due to astigmatism depends on the absolute difference between defocusU and defocusV and not on the ratio.

I volunteer to replace it in the CTF related protocols.

**Comment by delarosatrevin** [comment]
We should discuss about this...I'm not sure if replacing that could break
something. It doesn't make sense to add this difference instead of replace
the current ratio?

**Comment by rmarabini** [comment]
It is OK with me

**Comment by azazellochg** [comment]
I agree with @rmarabini. It's much easier to judge amount of astigmatism looking at the difference rather than smth like "1.05" ratio
